flake8_trio.py

```diff
@@ -79,3 +120,4 @@ def run(self) -> Generator[Tuple[int, int, str, Type[Any]], None, None]:
 
 
 TRIO100 = "TRIO100: {} context contains no checkpoints, add `await trio.sleep(0)`"
+TRIO101 = "TRIO101: {} never yield inside a nursery or cancel scope"
```

Comment:
```suggestion
TRIO101 = "TRIO101: yield inside a {} context is only safe when implementing a context manager - otherwise, it breaks exception handling"
```